src/components/gridHOC.jsx

```diff
@@ -25,6 +26,10 @@ type Props = {
   pageSize: number,
   cellComponent: Component<*>,
   editButtonLabel: string,
+  exportable: boolean,
+  exportData: Array<Object>,
+  exportButtonLabel: String,
+  fileName:string,
```

Comment:
can you make this `exportFileName` so it is clear that these props are all related to export?